### ISSUE TITLE
Api Docs: Document type

### ISSUE THREAD

**Issue by theduke** [apollostack/core-docs#234]
The Api Docs reference a `Document` type, which is linked.

But there is no documentation for it.

**Comment by helfer** [comment]
@theduke yeah, that's a bit unfortunate. That type actually comes from typed-graphql, and it refers to a GraphQL document. I don't know if there's any way for us to add that type to the docs (or if it even makes sense). I wish there was something like automatic documentation for all @types packages.
